src/metabase/models/dashboard.clj

```diff
@@ -216,11 +216,19 @@
 
 (defmethod revision/serialize-instance :model/Dashboard
   [_model _id dashboard]
-  (-> (apply dissoc dashboard excluded-columns-for-dashboard-revision)
-      (assoc :cards (vec (for [dashboard-card (dashcards dashboard)]
-                           (-> (apply dissoc dashboard-card excluded-columns-for-dashcard-revision)
-                               (assoc :series (mapv :id (dashboard-card/series dashboard-card)))))))
-      (assoc :tabs (map #(apply dissoc % excluded-columns-for-dashboard-tab-revision) (tabs dashboard)))))
+  (let [cards (or (:cards dashboard)
```

Comment:
optimization: only hydrates when needed

Reply:
renamed to dashcards https://github.com/metabase/metabase/pull/34947/commits/d542853d655500e2e2288514f83a1d6ec589179f